src/SDK/Language.php

```diff
@@ -106,6 +106,22 @@ protected function toCamelCase($str): string
         return $str;
     }
 
+    protected function toLowerCamelCase($str): string
+    {
+        // Normalize the string to decompose accented characters
+        $str = \Normalizer::normalize($str, \Normalizer::FORM_D);
+
+        // Remove accents and other residual non-ASCII characters
+        $str = preg_replace('/\p{M}/u', '', $str);
+
+        // Replace non-alphanumeric characters with a space
+        $str = preg_replace('/[^a-zA-Z0-9]+/', ' ', $str);
+        $str = ucwords(strtolower($str));
+        $str = lcfirst(str_replace(' ', '', $str));
+
+        return $str;
+    }
```

Comment:
I think we just need the `\strtolower` added to the existing `toCamelCase` function before the `\ucwords`.

Reply:
Done.